content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -133,79 +119,194 @@ Examples of common errors indicating that there is an issue with trusting a TLS/
+2. Test the field for other kind of actions - sometimes mirroring is only supported for appending / editing a field and not removing, double check it doesn't work in any case, if it does work in some cases, then it's more likely a feature request.
+
+## long-running server
+
+### Getting 400 error code when trying to pull from the long-running server
+
+This may occur due to docker being shut down without the server noticing. In order to reset the docker, go to integrations configuration page, disable the integration for a few seconds, and re-enable it. This action should attempt to kill the docker and re-deploy it.
+
+## Automations (command / playbook)
+
+### Not seeing any task outputs
+
+New playbooks are set to quiet-mode by default. Make sure to unquiet them.
+Alteratively, you can set the extend output/context option.
+![extend-context-example](../../../docs/doc_imgs/reference/extend-context-example.png)
+
+### Failing to obtain value from previous task
+
+Make sure the data type is configured as "From Previous Tasks" rather than "As value".
+
+### Command is failing as part of the script / playbook
 
-2. If the issue does not reproduce consistently:
+Double check the command execution from the CLI with the basic arguments, this should help you figure out if the issue is with the integration and the command itself, or there's some failure in the logic of the playbook/script flow.
 
-   - [Set the log level](https://xsoar.pan.dev/docs/reference/articles/troubleshooting-guide#integration-debug-logs) of the specific instance for more convenient tracking of the fetch logs over time.
-   - Keep track on the [Fetch History](https://xsoar.pan.dev/docs/reference/articles/troubleshooting-guide#fetch-history) of this instance. Consider temporarily setting the **fetch.history.size** server configuration to store more records.
+## TIM
 
+### Missing indicators in the threat intel page
+
+Sometimes indicators that were fetched cannot be seen in the threat intel page, if you encounter such an issue, double-check the exclusion list by heading to settings > object setup > indicators > exclusion list, and ensure the indicator is not a part of the list.
+
+### A mismatch between the number of fetched indicators and the number of new indicators in the threat intel page
+
+Sometimes some of the fetched indicators are already known to the system. In that case, the existing indicator will be modified rather than recreated. You can double-check the number of new/omitted/modified indicators in the logs.
+Example:
+
+```bash
+[TIM][Recorded Future Feed][Recorded Future Feed_hash] Feed fetch - createIndicators command - batch number 2 - processed 2000 indicators, 0 skipped, 100 modified, 144 new
+```
+
+### Failing to obtain indicators from 3rd party application using Taxii server
+
+Ensure the issue is not related to the 3rd party, attempt to obtain the indicators using postman to make sure the searched URL is indeed correct.
```

Comment:
is there a fetch debug-mode command we can tell them to run that would give us a curl? (that can then be imported into postman/run from cli)